src/app/authentication/layouts/default/default.component.scss

```diff
@@ -0,0 +1,49 @@
+@import '~bootstrap/scss/bootstrap';
```

Comment:
Needed to add this to be able to use Bootstrap's media breakpoint mixin

Reply:
It will fail in the production while running `ng build --prod`(can test in development machine too) as angular's default component style size limit is `4 KB`.  It will cross the limit as it loads all the bootstrap components. It will show `default.component.scss exceeded maximum budget. Budget 4 kB was not met by 138 kB with a total of 142 kB.` error

We can fix it importing only required component
```
@import "~bootstrap/scss/functions";
@import "~bootstrap/scss/variables";
@import "~bootstrap/scss/mixins/breakpoints";
```

Reply:
Thanks 🙌🏼